### PR TITLE
update build to workaround https://github.com/fsprojects/Paket/issues/1289

### PR DESCRIPTION
**Pull request by dsyme**
Workaround the above paket problem

### CODE REVIEW

File: paket.dependencies [hunk 1]
```diff
@@ -1,5 +1,6 @@
 source https://nuget.org/api/v2
 
+nuget Nuget.CommandLine
```
Comment (forki) [inline paket.dependencies]: do you need this?

Comment (dsyme) [inline paket.dependencies]: Only to actually push the package (or can you use paket for that?)  "build PublishPackage" didn't seem to actually push it to nuget but maybe it was missing the api key

Comment (forki) [inline paket.dependencies]: you need to pass the api key to the build.

I always have a (gitignored) release.cmd in my repos. You can see http://fsprojects.github.io/ProjectScaffold/release-process.html how it works.

Comment (dsyme) [inline paket.dependencies]: ok thanks

**Pull request merged**